Unity/Editor/InitVCRTForwarders.cs

```diff
@@ -32,18 +33,37 @@ static InitVCRTForwarders()
             return;
         }
 
+        List<string> searchFolders = new List<string>()
+        {
+            Application.dataPath,                                       // Assets folder
+            Path.GetFullPath(Path.Combine("Library", "PackageCache")),  // Library\PackageCache folder
+            Path.GetFullPath("Packages")                                // Pacakges folder
+        };
+
         // Find a representative VCRTForwarders binary - there should be only one.
-        var assets = AssetDatabase.FindAssets(Path.GetFileNameWithoutExtension(moduleName));
-        if (assets.Length != 1)
+        string dllDirectory = string.Empty;
+        for (int i = 0; i < searchFolders.Count; i++)
         {
-            Debug.LogError(string.Format("Failed to find single asset for {0}; found {1} instead!", moduleName, assets.Length));
+            string[] files = Directory.GetFiles(searchFolders[i], moduleName, SearchOption.AllDirectories);

```

Comment:
Assuming you accept the previous suggestion, you'll need this too:
```suggestion
            string[] files = Directory.GetFiles(searchFolder, moduleName, SearchOption.AllDirectories);
```